docs/tutorials/piaf.rst

```diff
@@ -82,6 +82,27 @@ the latter element could be accessed directly as follows:
           use either raw strings (using the `r` prefix, as in the example
           above) or escape each backslash as `\\\\\\\\`.
 
+.. _finding_attributes:
+
+************************************
+Finding attributes based on full path
+************************************
+
+To get the direct attribute based on the entire element/attributes path
+you can use the :any:`PIAFElement.attributes` method. You can provide a single string or list of strings with
+the full path and returns a list of attribute objects.
+
+.. code-block:: python
+
+    import PIconnect as PI
+
+    with PI.PIAFDatabase() as database:
+        attributes = database.attributes(r"Plant1\Outlet|Flow|PV", r"Plant1\Outlet|Flow|SP")
```

Comment:
The method accepts either a single string or a list of strings, but not multiple individual strings. So I think you are missing a pair of [].

Reply:
yep, fixed